dtfs-central-api/src/index.ts

```diff
@@ -6,4 +6,4 @@ initialiseCronJobScheduler(cronSchedulerJobs);
 
 const PORT = process.env.PORT || 5005;
 
-app.listen(PORT, () => console.info('✅ Central micro-service initialised on :%s', PORT));
+app.listen(PORT, () => console.info('✅ Central micro-service initialised on $s', PORT));
```

Comment:
We sure we want a `$` here?

Reply:
Good spot, this was accidental have rectified.